test-fastapi.sh

```diff
@@ -0,0 +1,24 @@
+#! /usr/bin/env bash
+
+set -x
+set -e
+
+if [ ! -d "./fastapi/" ]
+then
+    git clone https://github.com/tiangolo/fastapi.git
+fi
+cd fastapi
+git fetch --tags
+latest_tag_commit=$(git rev-list --tags --max-count=1)
+latest_tag=$(git describe --tags "${latest_tag_commit}")
+git checkout "${latest_tag}"
+pip install flit
+flit install
+SKIP_CYTHON=1 pip install -e ..
+
+# Remove temporary DB
+if [ -f ./test.db ]; then
+    rm ./test.db
```

Comment:
most make recipes just use `rm -f test.db` or you could just add this to the clean recipe and `.gitignore`? I'm fine with either.

Reply:
Perfect, added to `make clean`.